added telemetry examples (#217)
chargetrip/developers: content/API-Reference/5+Navigation/update-navigation.md

title: Mutate to update a navigation session
description: Update a navigation session by providing locations, telemetry and heading
order: 5
telemetryFields:
- state_of_charge
- tire_pressure
- vehicle_speed
- battery_voltage
- battery_current
- motor_rpm
- aux_power
- odometer
- current_weight
- occupants
- ...

telemetryDescriptions:
- Battery state of Charge [%] or [kWh]
- Average tire pressure of all wheels [Bar]
- Vehicle speed [kmph]
- Battery voltage [V]
- Battery current [A]
- RPM of the motor
- Auxiliary power consumption [kW]
- Mileage or Odometer reading [km]
- Active weight of the vehivle on top of curb weight [g]
- Pax currently in the vehicle

> - custom input fields can be added based on telemtry architecture

Reviewer: Fix

Reviewer: ```suggestion
- custom input fields can be added based on telemetry architecture
```

## Mutation

> In order to update a navigation session you need to provide the id and an array of `locations_data`. The other fields, `heading` and `telemetry` are optional but including them will result in better consumpotion predictions and more accurate routing.

Reviewer: Fix

Reviewer: ```suggestion
In order to update a navigation session you need to provide the id and an array of `locations_data`. The other fields, `heading` and `telemetry` are optional but including them will result in better consumption predictions and more accurate routing.
```

### Locations data heartbeat

You need to send at least one location in `locations_data` array. If you are using Mapbox as a data provider you will periodically receive updates that can be stored in a local cache. Alternatively, you could store a location every 5 or 10 seconds depending on the vehicle speed.

We recommend you to send the locations at least every 5 minutes.

### Telemetry data

Besides location data our API accepts telemetry data. This includes the `state_of_charge` of the vehicle. E.g. this allows you to send us a new value after charging at a station. Or you could send us intermittend SoC values to confirm the rolling consumption prediction. Telemetry data can either come from a user interface, an IVI system or a telemetry API. 

> Besides the `state_of_charge`, you can provide us with the average `tire_pressure` of all wheels of the vehicle and whether or not the vehicle is charging via the `is_charging` boolean. Besides these examples,there are many more accepted telemetry fields depending on the make and model of the vehicle. Below is a table with accepted telemetry fields given your telemtry architecture supports them. Chargetrip can easily add additional custom telemtry inputs; heatpumps, curb weight, refrigeration, PV etc. Please contact your account manager or contact sales@chargetrip.com

Reviewer: Fix

Reviewer: ```suggestion
Besides the `state_of_charge`, you can provide us with the average `tire_pressure` of all wheels of the vehicle and whether or not the vehicle is charging via the `is_charging` boolean. Besides these examples,there are many more accepted telemetry fields depending on the make and model of the vehicle. Below is a table with accepted telemetry fields given your telemetry architecture supports them. Chargetrip can easily add additional custom telemetry inputs; heat pumps, curb weight, refrigeration, PV etc. Please contact your account manager or contact sales@chargetrip.com
```
